chapters/1.md

```diff
@@ -1,100 +1,171 @@
-# Chapter 1 
-## Introduction
+# Chapter 1
 
-## An Introduction to Cryptocurrency and Privacy
+# Introduction
 
-Meet Leo and George. George would like to go on vacation, but he has not saved enough and has limited access to money. Leo, who is George’s best friend, is kind enough to lend George $2,500 to pay for a flight, hotel room, and food.
+## How can we send money to a friend?
 
-In traditional banking, in order for Leo to send George the $2,500, Leo would ask his bank to send the money to George’s bank. Leo would tell George that the transaction has taken place, and George would then be able to withdraw the needed $2,500 for his vacation. In this transaction, Leo and George have trusted their banks to manage the money.
+Meet Maria and her best friend George, who decide to meet up for a vacation. George’s budget is tight, so Maria offers to send some money to help pay for the flight, hotel room, and food.
```

Comment:
I still disagree with this opening example.  If the book is trying to appeal to a global audience, I find it hard to believe that many people will identify with this.  Especially as an opening example to the book, it should be realistic, or simpler so it is reachable to a larger audience.  Most people I know would lend money to their friends, but when it's "flight, hotel room, and food", it makes it sounds over the top.   Even removing "hotel room and food" would help I think.

Reply:
@wonal, I think you make a good point! We want to portray Monero as an everyday currency, and an enormous transaction like {books, hotels, flights} is probably not the best way to do that. I will change to something more relatable when I revisit the chapter.